pyproject.toml

```diff
@@ -1,3 +1,3 @@
 [build-system]
-requires = ["cython", "oldest-supported-numpy", "setuptools", "wheel"]
+requires = ["cython", "oldest-supported-numpy", "setuptools", "swig<4", "wheel"]
```

Comment:
#839 fixed our SWIG 4 issue

Reply:
Thanks. I'll remove the restriction and see what happens. As you suggested, it may also make sense to pin to a fixed version.